batch/batch_commands.go

```diff
@@ -151,14 +151,14 @@ func startBatchJob(c *cli.Context, req *workflowservice.StartBatchOperationReque
 		Namespace: namespace,
 		Query:     query,
 	})
-	if err != nil {
-		return fmt.Errorf("unable to count impacted workflows: %w", err)
+
+	var promptMsg string
+	if err == nil {
+		promptMsg = fmt.Sprintf("Will start a batch job operating on %v Workflow Executions. Continue? Y/N", color.Yellow(c, "%v", count.GetCount()))
+	} else {
+		promptMsg = fmt.Sprintf("Will start a batch job on all Workflow Executions that match the %s. Continue? Y/N", color.Yellow(c, "%v", query))
```

Comment:
Hmm.. I didn't know we were prompting here.
Is there a way to disable the prompt?

Reply:
it can be disabled if the `--yes` flag is passed